src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/on_system_capability_updated_notification.cc

```diff
@@ -0,0 +1,73 @@
+#include "sdl_rpc_plugin/commands/mobile/on_system_capability_updated_notification.h"
+#include "application_manager/application_manager.h"
+#include "application_manager/helpers/application_helper.h"
+#include "application_manager/message_helper.h"
+#include "sdl_rpc_plugin/extensions/system_capability_app_extension.h"
+#include "smart_objects/enum_schema_item.h"
+
+namespace sdl_rpc_plugin {
+using namespace application_manager;
+namespace commands {
+namespace mobile {
+
+OnSystemCapabilityUpdatedNotification::OnSystemCapabilityUpdatedNotification(
+    const app_mngr::commands::MessageSharedPtr& message,
+    app_mngr::ApplicationManager& application_manager,
+    app_mngr::rpc_service::RPCService& rpc_service,
+    app_mngr::HMICapabilities& hmi_capabilities,
+    policy::PolicyHandlerInterface& policy_handler)
+    : CommandNotificationImpl(message,
+                              application_manager,
+                              rpc_service,
+                              hmi_capabilities,
+                              policy_handler) {}
+
+OnSystemCapabilityUpdatedNotification::
+    ~OnSystemCapabilityUpdatedNotification() {}
+
+void OnSystemCapabilityUpdatedNotification::Run() {
+  LOG4CXX_AUTO_TRACE(logger_);
+
+  // TODO, figure out how to handle unknown enums for SystemCapabilityType
```

Comment:
I dont think core will have to handle unknown enums in SystemCapabilityType. Future unknown app services will be covered under the `APP_SERVICES` system capability type. This assumption needs clarification though. I believe the servicesSupported parameter in appServiceCapabilities should be changed to type string.

Reply:
Understood. I will remove the TODO comment